vendir-template.yml

```diff
@@ -0,0 +1,19 @@
+#! Copyright 2021 VMware, Inc.
+#! SPDX-License-Identifier: Apache-2.0
+#@ load("@ytt:data", "data")
+apiVersion: vendir.k14s.io/v1alpha1
+kind: Config
+minimumRequiredVersion: 0.12.0
+directories:
+  - path: config/release
+    contents:
+    - path: .
+      http:
+        url: #@ "https://storage.googleapis.com/beam-releases/kpack-release-yaml/release-{}.yaml".format(data.values.version)
```

Comment:
Is this a stable location for releases?

Reply:
it should be since that's where tbs gets it from, but im happy to start pushing to an alternate location as well